src/components/Combobox/Combobox.jsx

```diff
@@ -1,5 +1,5 @@
 /* eslint-disable jsx-a11y/click-events-have-key-events */
-import React, { useRef, useState, forwardRef, useMemo, useCallback } from "react";
+import React, { useRef, useState, forwardRef, useMemo, useCallback, useEffect } from "react";
```

Comment:
right now I don't have a way to test it in combobox because we did not implement any infra for testing visual focus there yet